src/renault_api/kamereon/helpers.py

```diff
@@ -38,13 +40,42 @@ def update_schedule(schedule: models.ChargeSchedule, settings: dict[str, Any]) -
                 )
 
 
-def create_schedule(
+def update_hvac_schedule(
+    schedule: models.HvacSchedule, settings: dict[str, Any]
+) -> None:
+    """Update HVAC schedule."""
+    if "activated" in settings:
+        schedule.activated = settings["activated"]
+    for day in DAYS_OF_WEEK:
+        if day in settings:
+            day_settings = settings[day]
+
+            if day_settings is None:
+                setattr(schedule, day, None)
+            elif day_settings:  # pragma: no branch
+                ready_at_time = day_settings["readyAtTime"]
+
+                setattr(
+                    schedule,
+                    day,
+                    models.HvacDaySchedule(day_settings, ready_at_time),
+                )
+
+
+def create_charge_schedule(
```

Comment:
Please keep a copy of `create_schedule` and mark it as deprecated, replaced by `create_charge_schedule` to avoid a breaking change.